pkg/helper/client.go

```diff
@@ -188,10 +188,16 @@ func ApplyDesiredState(nodeNetworkState *nmstatev1alpha1.NodeNetworkState) (stri
 		}
 	}
 
+	err = exec.Command("ping", "-c", "3", "8.8.8.8").Run()
```

Comment:
Please hide this in a helper function, pass only the IP address.

Also, the IP should be read from our config map.

Reply:
Yep that was the next step, pushed it to see how it goes at stdci.